code/modules/mob/living/carbon/xenomorph/castes/spiderling/spiderling.dm

```diff
@@ -135,9 +133,6 @@
 		return
 	if(isliving(atom_to_walk_to))
 		var/mob/living/victim = atom_to_walk_to
```

Comment:
Check does nothing any more since you've taken the lines after out.

Reply:
You think I should remove the whole check ?

Reply:
Or put them back and check for DEATH stat specifically to shortcut some proc calls, either way.